1.8/usage/service-discovery/dns-overview.md

```diff
@@ -0,0 +1,65 @@
+---
+post_title: DNS Quick Reference
+nav_title: DNS Reference
+menu_order: 20
+---
+
+DC/OS offers a lot when it comes to DNS.
+This quick reference provides a summary of the available options.
+
+# Example Setup
+
+To make things more concrete, we'll use this example application:
+* (Group) `outergroup` > (Group) `subgroup` > (Service Name) `myapp`
```

Comment:
It's the group hierarchy, I can experiment with nesting bullets to see if it's clearer.